src/objects/avatar/Avatar.ts

```diff
@@ -606,6 +607,14 @@ export class Avatar extends RoomObject implements IMoveable, IScreenPositioned {
   private _stopWalking() {
     this._walking = false;
     this._updateAvatarSprites();
+
+    if (this._onStopWalkingCallback) {
+      this._onStopWalkingCallback();
```

Comment:
In `shroom` there are two different types of movement. These are:

- `move` invoked with `avatar.move` - movement without a walking animation
- `walk` invoked with `avatar.walk` - movement with a walking animation

Right now, this code only handles the `walk` movement. Do you think it would make sense to create a unified callback, which is called whenever the movement animation stops, most likely invoked within the the following function?

https://github.com/jankuss/shroom/blob/3c334982f37300ee291f4d970344950b5e1ebfa4/src/objects/avatar/Avatar.ts#L427

And then call the callback `onStopMovement`?

Reply:
Yes it makes sense, i will try to refactor it